src/providers/defProvider.ts

```diff
@@ -43,6 +43,20 @@ export class DefProvider implements vscode.DefinitionProvider {
             );
         }
 
+        // "get hotkey" must be below "get label"!
```

Comment:
When you say "below", do you mean "called after"? Can we reword this one for clarity?